### PR TITLE
create pairing_lokum.py

### PR DESCRIPTION
**Pull request by onurinanc**
I'll create a submodule for pairing.py, pairing_cp6_782.py, test_pairing.py and test_pairing_cp6_782.py later. It's a commit and PR to show pairing_lokum.py

### CODE REVIEW

**Comment by Mikerah** [comment]
Let's go 🚀

**Review by Mikerah — CHANGES_REQUESTED** [review]
- Add more detailed comments in the sage code in order to better guide not only yourself but other readers of the code on how to implement this from scratch
- Shouldn't the actual cairo code be in the starkacy repo?

**Comment by onurinanc** [comment]
- For the sage code, I'll add more guide
- For the Cairo code, now I'll open a new branch in Starkacy and add the code in there

File: pairing_cp6_782.py
Comment (Mikerah) [inline pairing_cp6_782.py:1]: Was this file changed from the original one? If so then, we should specify which parts have been changed and maybe change the names of the functions to reflect our own curve

**Review by Mikerah — CHANGES_REQUESTED** [review]

File: pairing_cp6_782.py
Comment (onurinanc) [reply pairing_cp6_782.py:1]: No, its the same file

**Review by Mikerah — APPROVED** [review]

**Pull request merged**